### PR TITLE
Add intial build instructions to the README

### PR DESCRIPTION
**Pull request by Jalle19**
This way people don't have to go hunting down the forums. I'll try to add Windows instructions as well.

@adamsutton can you try this?

### CODE REVIEW

**Comment by adamsutton** [comment]
@Jalle19 thanks for this :) only suggestion I'd have is to use generic/relative paths rather than your own specific ones.

**Comment by Jalle19** [comment]
@adamsutton updated. I dropped `make install` because for some strange reason it wasn't needed anymore. It didn't use to be either but when I tried compiling the first time it didn't put the addon files in the correct directory.

Can you try again to see if the commands are sufficient?

File: README.md [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+This is a [Kodi] (http://kodi.tv) PVR addon for connecting to a (tvheadend)[https://tvheadend.org] backend.
+
+## Build instructions
+
+### Linux
+
+The following instructions assume you check out all repositories in a directory named `Projects` in your home 
+directory. cmake doesn't recognize `~` which is why `$HOME` is used when referring to your home directory.
+
+In addition to standard build tools you need to have the following packages installed: `kodi-addon-dev kodi-pvr-dev`.
```
Comment (adamsutton) [inline README.md]: I have neither of these installed, doesn't seem to be an issue. Unless these are virtual packages which just install other packages I already have?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
+### Linux
+
+The following instructions assume you check out all repositories in a directory named `Projects` in your home 
+directory. cmake doesn't recognize `~` which is why `$HOME` is used when referring to your home directory.
+
+In addition to standard build tools you need to have the following packages installed: `kodi-addon-dev kodi-pvr-dev`. 
+These packages are available from Kodi's PPA repositories.
+
+1. `cd ~/Projects`
+2. `git checkout https://github.com/xbmc/xbmc.git`
```
Comment (adamsutton) [inline README.md]: git clone

File: README.md [hunk 3]
```diff
@@ -1,7 +1,28 @@
...
+3. `git checkout https://github.com/kodi-pvr/pvr.hts.git`
+4. `cd pvr.hts && mkdir build && cd build`
+5. `cmake -DADDONS_TO_BUILD=pvr.hts -DADDON_SRC_PREFIX=/home/negge/Projects -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=/home/negge/Projects/xbmc/addons -DPACKAGE_ZIP=1 /home/negge/Projects/xbmc/project/cmake/addons/`
```
Comment (adamsutton) [inline README.md]: cmake -DADDONS_TO_BUILD=pvr.hts -DADDON_SRC_PREFIX=../.. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=../../xbmc/addons -DPACKAGE_ZIP=ON ../../xbmc/project/cmake/addons

No need to refer to abs paths that won't be valid.

File: README.md [hunk 4]
```diff
@@ -1,7 +1,28 @@
...
+
+### Linux
+
+The following instructions assume you check out all repositories in a directory named `Projects` in your home 
+directory. cmake doesn't recognize `~` which is why `$HOME` is used when referring to your home directory.
+
+In addition to standard build tools you need to have the following packages installed: `kodi-addon-dev kodi-pvr-dev`. 
+These packages are available from Kodi's PPA repositories.
+
+1. `cd ~/Projects`
```
Comment (adamsutton) [inline README.md]: I'd drop this and the text about it. People should be capable of knowing where they want to put code if they're going to be building stuff.

Not that fussed, this is just my personal opinion.

File: README.md [hunk 5]
```diff
@@ -1,7 +1,28 @@
...
+2. `git checkout https://github.com/xbmc/xbmc.git`
+3. `git checkout https://github.com/kodi-pvr/pvr.hts.git`
+4. `cd pvr.hts && mkdir build && cd build`
+5. `cmake -DADDONS_TO_BUILD=pvr.hts -DADDON_SRC_PREFIX=/home/negge/Projects -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=/home/negge/Projects/xbmc/addons -DPACKAGE_ZIP=1 /home/negge/Projects/xbmc/project/cmake/addons/`
+6. `make`
```
Comment (adamsutton) [inline README.md]: For reasons that I don't yet understand, I don't get a zip file output.

File: README.md [hunk 6]
```diff
@@ -1,7 +1,28 @@
...
+
+The addon files will be placed in `/home/negge/Projects/xbmc/addons` so if build Kodi from source and run it directly
```
Comment (adamsutton) [inline README.md]: Remove ref to /home/negge

**Comment by Jalle19** [comment]
@adamsutton updated

File: README.md [hunk 7]
```diff
@@ -1,7 +1,21 @@
 [![Coverity Scan Build Status](https://scan.coverity.com/projects/5120/badge.svg)](https://scan.coverity.com/projects/5120)
 
-# Tvheadend HTSP PVR
-Tvheadend HTSP PVR client addon for [Kodi] (http://kodi.tv)
+# Tvheadend PVR addon for Kodi
+
+This is a [Kodi] (http://kodi.tv) PVR addon for connecting to a (tvheadend)[https://tvheadend.org] backend.
```
Comment (hudokkow) [inline README.md]: You should use [tvheadend] followed by (https://tvheadend.org) or links look funny.
See https://raw.githubusercontent.com/xbmc/xbmc/master/README.md for reference

Comment (Jalle19) [inline README.md]: You're right

**Pull request merged**